CIP-????/README.md

```diff
@@ -0,0 +1,192 @@
+
+Note that the serialisation format of the ledger state is unspecified and left as an
+implementation detail (unlike the format of blocks).
+
+### The ledger-script interface
+
+The ledger and Plutus scripts have a common interface, described in [CIP-35].
+CIPs relating to this inteface are relevant to both the ledger and to the Plutus CIP categories.
+
+## Specification
+
+### What merits a ledger CIP?
+
+The criterion for deciding if a change to the ledger merits a CIP is as follows:
+changes to the ledger require going through the CIP process whenever
+every implementation of the Cardano ledger needs to be standardized on the details.
+
+Bug fixes are an exception to this criterion, they do not merit a CIP except in the case
+that the fix is substantially complicated.
+A "bug fix" is a change to behavior where:
+- The implemented behavior does not match the specification; or
+- The specified behavior is clearly wrong (in the judgment of relevant experts)
+
+Serialization changes are another possible exception to the criterion.
+Many serialization changes can be handled as a part of the normal development process
+without the need for a CIP.
+Dramatic changes to the serialization, however, may benefit from the CIP process.
+
+The ledger rules MUST be standardized in order for consensus to be maintained,
+but things like the ledger events are more open to debate.
+
+Changes to the protocol parameter values do not require a CIP since they are
+a governance issue (see [CIP-1694]).
+
+### Expectations for ledger CIPs
+
+* Familiarity with the existing ledger specifications is required to propose changes to the ledger.
+* The CIP specifications for ledger CIPs must be sufficiently detailed for inclusion in
+  a formal ledger specification.
+* Some changes may require getting an expert opinion.
+  For example, changes to the incentives model (such as changing the reward calculation)
+  should include a proper game theoretic analysis.
+
+## Rationale
+
+### There is only one implementation, why limit the scope of ledger CIPs in this way?
+
+Even though there is currently only one implementation, this provides us with a clear
+definition of what is essential to the ledger.
+It also provides a clear path for future implementations.
+
+### Why is the specification vague about the role of ledger events in the CIP process?
+
+This decision should be left to the community as more use cases emerge.
+
+### Why is familiarity with the formal specifications required?
+
+It is not always clear which seemingly small details can make a large difference
+to the many consumers of the ledger.
+It is better that the CIP process achieve consensus on all the details than for
+these decisions to be made during the implementation phase.
+
+
+[CIP-35]: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0035
+[CIP-59]: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0059
+[CIP-1694]: https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694
```

Comment:
... and another obligatory section: your chose software licensing model and/or link, as per https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#licensing